src/features/training-program/training-program.tsx

```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
     courseName.includes('badge') ? 'aws fundamentals' : courseName,
     type,
   );
-
   const course = data as Course;
 
   const { title, content, image } = contentMap[courseName];
 
   const buttonLabel = courseName === 'react ru' ? 'Записаться' : 'Register';
 
+  const filteredContent =
+    type === 'Pre-school'
+      ? content.filter((component) => component.type.name !== 'Subtitle')
```

Comment:
Could you leave a comment inside `Subtitle.tsx` letting us know that if the component name is going to be changed, it should be updated here as well?